src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
-body {
+@import url(./container.scss);
+@import url(./card.scss); 
+@import url(./header.scss);
+@import url(./stars.scss);
+
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
you shouldn't reset indentations for the * selector

Reply:
i'm fix it, please check my work))